product_attribute_global_item_code/__openerp__.py

```diff
@@ -0,0 +1,40 @@
+
+# -*- coding: utf-8 -*-
+
+##############################################################################
+#
+#    Copyright (C) All Rights Reserved 2015 Akretion
+#    @author Abdessamad HILALI <abdessamad.hilali@akretion.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+{
+    'name': "Product",
+    'description': """
+            Global Product Code
+    """,
+    'author': "Akretion",
+    'website': "http://www.Akretion.com",
+    'license': 'AGPL-3',
+    'category': 'Product',
+    'version': '0.1',
+    'depends': ['product', 'product_attribute_code'],
+    'data': [
+        'product_view.xml'
+    ],
+    'demo': [
+        ],
+}
```

Comment:
cut image below, to make it more readable, reduce large size to be more readable